src/data/set/intervals/unordered_interval.lean

```diff
@@ -139,15 +159,24 @@ def interval_oc : α → α → set α := λ a b, Ioc (min a b) (max a b)
 -- Below is a capital iota
 localized "notation `Ι` := set.interval_oc" in interval
 
-lemma interval_oc_of_le (h : a ≤ b) : Ι a b = Ioc a b :=
+@[simp] lemma interval_oc_of_le (h : a ≤ b) : Ι a b = Ioc a b :=
```

Comment:
This shouldn't be simp - for some applications (like the interval integrals library) the unordered interval is the more important one, and shouldn't be simped away